PWGJE/EMCALJetTasks/AliEmcalJetTask.h

```diff
@@ -180,7 +183,9 @@ class AliEmcalJetTask : public AliAnalysisTaskEmcal {
   Double_t               fGhostArea;              ///< ghost area
   Double_t               fTrackEfficiency;        ///< artificial tracking inefficiency (0...1)
   TObjArray             *fUtilities;              ///< jet utilities (gen subtractor, constituent subtractor etc.)
-  Bool_t                 fTrackEfficiencyOnlyForEmbedding; ///<tituent Apply aritificial tracking inefficiency only for embedded tracks
+  Bool_t                 fTrackEfficiencyOnlyForEmbedding; ///< Apply aritificial tracking inefficiency only for embedded tracks
+  TF1                    fTrackEfficiencyFunction;///< Function that describes the artificial tracking efficiency to be applied on top of the nominal tracking efficiency, as a function of track pT
+  Bool_t                 fApplyArtificialTrackingEfficiency; ///< Flag to apply artificial tracking efficiency
```

Comment:
Can you add a bit more documentation somewhere (maybe into doxygen)? For example, the information that you include in the PR seems useful to have somewhere near the code. It doesn't need to be super extensive - I just think it would be good to add something.

I'd suggest we either start adding more extensive documentation into `AliEmcalJetTask` or perhaps just add (labeled) section [here](http://alidoc.cern.ch/AliPhysics/master/READMEjetfw.html)?

Reply:
I added to the standalone documentation pages, since there are now two ways to apply this track efficiency it is not very conducive to explaining in the class itself.

Reply:
Sounds great - thanks!